### PR TITLE
Update social accounts links

### PR DESCRIPTION
**Pull request by potiuk**
We have now Bluesky account as well, and Twitter is now X.

---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/contributing-docs/05_pull_requests.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Review by eladkal — APPROVED** [review]

**Review by kaxil — APPROVED** [review]

File: generated/PYPI_README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ PROJECT BY THE `generate-pypi-readme` PRE-COMMIT. YOUR CHANGES HERE WILL BE AUTO
 [![PyPI - Downloads](https://img.shields.io/pypi/dm/apache-airflow)](https://pypi.org/project/apache-airflow/)
 [![Artifact HUB](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/apache-airflow)](https://artifacthub.io/packages/search?repo=apache-airflow)
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
-[![Twitter Follow](https://img.shields.io/twitter/follow/ApacheAirflow.svg?style=social&label=Follow)](https://twitter.com/ApacheAirflow)
+[![Twitter Follow](https://img.shields.io/twitter/follow/ApacheAirflow.svg?style=social&label=Follow)](https://x.com/ApacheAirflow)
```
Comment (kaxil) [inline generated/PYPI_README.md:35]: X Follow?

File: pyproject.toml [hunk 2]
```diff
@@ -152,7 +152,10 @@ Homepage = "https://airflow.apache.org/"
 "Release Notes" = "https://airflow.apache.org/docs/apache-airflow/stable/release_notes.html"
 "Slack Chat" = "https://s.apache.org/airflow-slack"
 "Source Code" = "https://github.com/apache/airflow"
-Twitter = "https://twitter.com/ApacheAirflow"
+X = "https://twitter.com/ApacheAirflow"
```
Comment (aritra24) [inline pyproject.toml]: x.com?

**Review by aritra24 — APPROVED** [review]

File: generated/PYPI_README.md (hunk 1 above)
Comment (potiuk) [reply generated/PYPI_README.md:35]: nope... does not work this way :( . They shields.io is a little out-dated, not too maintained for the new social networks and X is still twitter there :(

File: pyproject.toml (hunk 2 above)
Comment (potiuk) [reply pyproject.toml]: yep.

Comment (potiuk) [inline pyproject.toml]:
```suggestion
X = "https://x.com/ApacheAirflow"
```

**Comment by ryanahamilton** [comment]
I'm curious if Apache allows adding a DNS record so a "verified" `@airflow.apache.org` handle [could be used](https://bsky.social/about/blog/4-28-2023-domain-handle-tutorial) on Bluesky?

**Comment by potiuk** [comment]
> I'm curious if Apache allows adding a DNS record so a "verified" `@airflow.apache.org` handle [could be used](https://bsky.social/about/blog/4-28-2023-domain-handle-tutorial) on Bluesky?

Yeah. Good idea. I saw that option yesterday and want to test it for my own account first. I **think** that might require JIRA issue to create with infra. So if you would like to open such issue - maybe you could? @ryanahamilton ? the necessary information on all the infra services is here: https://infra.apache.org/

**Pull request merged**